src/pages/home/Home.jsx

```diff
@@ -1,70 +1,85 @@
-                <img
-                  src={bCategory.imgSrc}
-                  alt={bCategory.imgAlt}
-                  className="responsive-img opaque"
-                />
-                <a className="text-overlay-card" href={bCategory.link}>
-                  {bCategory.category}
-                </a>
-              </div>
+  const [categories, setCategories] = useState([]);
+
+  return (
+    <>
+      <main className="flex-col">
+        {/* <!-- block-1 --> */}
+        <div className="display-block">
+          <div className="flex-row landscape-overcard">
+            <img
+              src="/assets/images/sofaBeige.webp"
+              alt="living room decor"
+              className="responsive-img"
+            />
+            {/* <!-- over-card --> */}
+            <div className="overlay-solid-card med-text">
+              Welcome to your one stop shop for elegant and chic Decor.
             </div>
-          );
-        })}
-      </div>
-      {/* <!-- end of block-2 --> */}
+          </div>
+        </div>
+        {/* <!-- end of block-1 --> */}
 
-      {/* <!-- block-3 --> */}
-      <div className="display-block flex-row-wrap flex-center">
-        {categories.map((category) => {
-          return (
-            <div className="category-card" key={category.category}>
-              <a className="text-dec-none" href={category.link}>
-                <div className="inner-category-card card-scale">
+        {/* <!-- block-2 --> */}
+        <div className="display-block flex-row">
+          {broadCategories.map((bCategory) => {
+            return (
+              <div className="category-overlay-card" key={bCategory.category}>
+                <div className="inner-category-overlay card-scale flex-center">
                   <img
-                    src={category.imgSrc}
-                    alt={category.imgAlt}
-                    className="responsive-img"
+                    src={bCategory.imgSrc}
+                    alt={bCategory.imgAlt}
+                    className="responsive-img opaque"
                   />
-                  <p className="mg-xs text-center">{category.category}</p>
+                  <a className="text-overlay-card" href={bCategory.link}>
+                    {bCategory.category}
+                  </a>
                 </div>
-              </a>
-            </div>
-          );
-        })}
-      </div>
-      {/* <!-- end of block-3 --> */}
-    </main>
-  </>
-);
+              </div>
+            );
+          })}
+        </div>
+        {/* <!-- end of block-2 --> */}
+
+        {/* <!-- block-3 --> */}
+        <div className="display-block flex-row-wrap flex-center">
+          {categories.map((category) => {
+            return (
+              <div className="category-card" key={category.category}>
+                <a className="text-dec-none" href={category.link}>
+                  <div className="inner-category-card card-scale">
```

Comment:
instead of using anchor tag Link should be used anchor tag should be used for absolute paths.
You can read about react absolute and relative path and also links and anchor

Reply:
i haven't finalised the links yet i have plans to change it afterwards when i'm done with static pages so i will remove a tag from there.